core/api/src/domain/wallet-invoices/wallet-invoice-builder.ts

```diff
@@ -143,7 +143,7 @@ export const WIBWithAmount = (state: WIBWithAmountState): WIBWithAmount => {
       return new InvalidWalletInvoiceBuilderStateError()
     }
 
-    const walletInvoice: WalletInvoice = {
+    const walletInvoice = {
```

Comment:
```suggestion
    const walletInvoice: WalletInvoice = {
```

we can keep the type if we don't make paymentRequest optional here https://github.com/GaloyMoney/galoy/pull/3333/files#r1355431350

(note: we can still keep the type optional at the db layer)

Reply:
After a migration, I will make the type non-optional and then we can use wallet invoice.